From: Hector Cao <hector.cao@canonical.com>
Date: Wed, 17 Jul 2024 08:52:39 +0200
Subject: [PATCH] Enable private home directories by default

Before the version 1:4.15.3-2, Debian lets the
HOME_MODE variable unset in the debian/login.defs file
Ubuntu uncomments the variable and sets the HOME_MODE
to 0750
But with this new version, Debian uncomment the HOME_MODE
variable that has the value 0700
This patch tries to keep the Ubuntu current behavior by
overriding the HOME_MODE from Debian to have the value 0750
---
 etc/login.defs | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/etc/login.defs b/etc/login.defs
index 91d3ec4..bfb81af 100644
--- a/etc/login.defs
+++ b/etc/login.defs
@@ -85,7 +85,7 @@ KILLCHAR	025
 
 # HOME_MODE is used by useradd(8) and newusers(8) to set the mode for new
 # home directories.
-HOME_MODE	0700
+HOME_MODE	0750
 
 #
 # Password aging controls:
